### PR TITLE
Update to SPP8 and API3.0.0-ALPHA7

### PR DESCRIPTION
**Pull request by tarik02**
- Version bump: 3.0.0-ALPHA7
 - Update to SPP8 and API 3.0.0-ALPHA7
 - Temponary removed loading screen
 - Need to merge https://github.com/iTXTech/Nemisys/pull/136
 - Reformatted code, removed Genisys headers
 - Used strict types
 - Performance improvements
 - Compatibility improvements

### CODE REVIEW

**Comment by PeratX** [comment]
Just add ```.idea/``` to your .gitignore is OK.

**Comment by tarik02** [comment]
@PeratX why remove idea? There are special gitignore that saves only shared info.
Strict types improve performance. I fixed everything with it and it works perfectly.

**Comment by dktapps** [comment]
Strict types are love, they make it much more difficult to make stupid mistakes. Crash > unexpected behaviour.

**Comment by tarik02** [comment]
#34

File: src/synapsepm/network/protocol/spp/Info.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?php
+declare(strict_types=1);
+namespace synapsepm\network\protocol\spp;
+
+class Info {
+	const CURRENT_PROTOCOL = 8;
+	const PROTOCOL_MAGIC = 0xbabe;
```
Comment (ghost) [inline src/synapsepm/network/protocol/spp/Info.php:7]: babe? =))))))))))))

In ro: babe = grandmother =)))

Comment (tarik02) [reply src/synapsepm/network/protocol/spp/Info.php:7]: Oh, it's not mine. It's a part of Synapse Proxy Protocol

**Comment by MisteFr** [comment]
working with Pocketmine @Tarik02 ?

**Comment by tarik02** [comment]
@Misteboss yes

**Pull request merged**